### PR TITLE
Template for dataset explorer block

### PR DESCRIPTION
**Pull request by xleafx**

### CODE REVIEW

File: modules/quanthub_datasetexplorer/quanthub_datasetexplorer.module [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Implements hook_theme().
+ */
+function quanthub_datasetexplorer_theme() {
+  return [
+    'datasetexplorer_block' => [
+      'template' => 'datasetexplorer-block',
```
Comment (drudoi) [inline modules/quanthub_datasetexplorer/quanthub_datasetexplorer.module]: This definition isn't necessary

File: modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php [hunk 2]
```diff
@@ -45,80 +45,6 @@ public static function create(ContainerInterface $container) {
     );
   }
 
-  /**
-   * Build Slices block.
-   */
-  public function slices(Request $request) {
```
Comment (drudoi) [inline modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php:51]: Please ask Ilya about should we delete it from the profile.

File: modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php [hunk 3]
```diff
@@ -141,9 +67,6 @@ public function explorer(Request $request) {
 
     $build = [
       '#type' => 'container',
```
Comment (drudoi) [inline modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php]: Please return same to the block
```
'#theme' => 'datasetexplorer_block',
'#attached' => ...,
```

**Review by drudoi — CHANGES_REQUESTED** [review]

File: modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php (hunk 2 above)
Comment (illya-havsiyevych) [reply modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php:51]: let's remove /slices

File: modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php [hunk 4]
```diff
@@ -45,80 +45,6 @@ public static function create(ContainerInterface $container) {
...
-
-    $build['element-content']['#attached']['drupalSettings']['mode'] = 'slices';
-    $build['element-content']['#attached']['drupalSettings']['query'] = $request->query->all();
-    return $build;
-  }
-
-  /**
-   * Build pivot block.
-   */
-  public function pivot(Request $request) {
```
Comment (illya-havsiyevych) [inline modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php:88]: let's keep /pivot

Comment (drudoi) [reply modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php:88]: Decided to remove too.

**Review by drudoi — APPROVED** [review]

**Pull request merged**